### PR TITLE
feature-login-base-structure

### PR DESCRIPTION
**Pull request by Novak-Fenocchio**
Login of discord.

### CODE REVIEW

File: html/enter.html [hunk 1]
```diff
@@ -1,88 +1,88 @@
...
+                    <a class="nav-link active" href="register.html">Regito</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="enter.html">Entrar</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="index.html">home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Sair</a>
```
Comment (JonathanCR1) [inline html/enter.html:58]: Use english. :P

**Comment by jonasnapoles** [comment]
done

**Comment by jonasnapoles** [comment]
I have not seen the modifications, it seems all the same.

**Review by jonasnapoles — COMMENTED** [review]
I have not seen the modifications, it seems all the same.

**Pull request merged**